DevEdu.API/DevEdu.API/Controllers/CourseController.cs

```diff
@@ -146,6 +146,26 @@ public List<CourseTopicOutputModel> SelectAllTopicsByCourseId(int courseId)
             return _mapper.Map<List<CourseTopicOutputModel>>(list);
             
         }
+        // api/course/{courseId}
+        [HttpPut("{courseId}/update")]
```

Comment:
please rename to "{courseId}/program" (and the comment on 149 line as well)

Reply:
fixed